### ISSUE TITLE
show config topic field blank when code uploaded from platformio

### ISSUE THREAD

**Issue by nilava** [xoseperez/espurna#1762]
The show config in HASS is not working properly when i am uploading code from platform io instead of using the prebuilt firmware for nodemcu lolin.

```
switch:
  - name: ESPURNA_1F7B48
    platform: mqtt
    state_topic: 
    command_topic: 
    payload_on: 1
    payload_off: 0
    availability_topic: 
    payload_available: 1
    payload_not_available: 0
 ```

This is what i am getting where as you can see state_topic, command_topic and availability_topic is blank

**Comment by xoseperez** [comment]
What is your MQTT configuration?

**Comment by nilava** [comment]
What is my MQTT Config as in?

**Comment by xoseperez** [comment]
Can you copy paste here the output for the `keys` command in the debug console (removing private data)?

**Comment by jyz0501** [comment]
payload_on: 1
payload_off: 0

It should be written like this

    payload_on: "ON"
    payload_off: "OFF"


The  show  config  is  wrong！

**Comment by mcspr** [comment]
@jyz0501 please see #1883 
If you use current `dev` build and you do see `ha.config` outputting mqtt toic, I believe this is no longer a bug, as it should've been fixed by the #1755
